pkg/nodeconfig/nodeconfig.go

```diff
@@ -664,13 +664,34 @@ func (nc *nodeConfig) UpdateKubeletClientCA(contents []byte) error {
 	return nil
 }
 
+// bootstrap starts all required services to bootstrap a node object
+func (nc *nodeConfig) bootstrap() error {
+	if err := nc.Windows.Prepare(); err != nil {
+		return err
+	}
+	saCA, saToken, err := getWICDCertAndToken(nc.k8sclientset)
```

Comment:
Please create a struct for the `saCA, saToken` as you are passing this pair around a lot.